Sources/ComposableStoreKit/Live.swift

```diff
@@ -113,7 +113,7 @@ private class Observer: NSObject, SKPaymentTransactionObserver {
   }
 
   func paymentQueueRestoreCompletedTransactionsFinished(_ queue: SKPaymentQueue) {
-    self.subscriber.send(.restoreCompletedTransactionsFinished)
+    self.subscriber.send(.restoreCompletedTransactionsFinished(transactionCount: queue.transactions.count))
```

Comment:
What do you think about returning the transactions here instead? We already have a `PaymentTransaction` type we own that you could use to wrap the raw value:

https://github.com/pointfreeco/isowords/blob/f77002115e1344d945345ac17c3e3f4ffe0b753b/Sources/ComposableStoreKit/Client.swift#L66

Reply:
Great call, updated in https://github.com/pointfreeco/isowords/pull/85/commits/0f9a6cd4156caa5ed3e5cd4c138b2dc35c04aa56 ✅